test/e2e/extensions.test.ts

```diff
@@ -1,7 +1,7 @@
-import * as path from "path"
 import { test as base } from "@playwright/test"
-import { describe, test, expect } from "./baseFixture"
+import * as path from "path"
 import { getMaybeProxiedCodeServer } from "../utils/helpers"
+import { describe, test, expect } from "./baseFixture"
```

Comment:
`yarn lint` did this - not sure why it didn't catch it before

Reply:
Quite strange indeed!